packages/gatsby-source-filesystem/src/gatsby-node.js

```diff
@@ -11,218 +11,159 @@ const { createId, createFileNode } = require(`./create-file-node`)
  * On the latter, this solves the problem where if you call createNode for the
  * same File node in quick succession, this can leave Gatsby's internal state
  * in disarray causing queries to fail. The latter state machine tracks when
- * Gatsby is "processing" a node update or when it's "idle". If updates come in
+ * Gatsby is "busy" with a node update or when it's "idle". If updates come in
  * while Gatsby is processing, we queue them until the system returns to an
  * "idle" state.
  */
+
 const fsMachine = Machine({
-  key: "emitFSEvents",
+  key: `emitFSEvents`,
   parallel: true,
-  strict: true,
   states: {
-    CHOKIDAR: {
-      initial: `CHOKIDAR_NOT_READY`,
+    SRC_FS: {
```

Comment:
Why did you change these names? I'm not sure what your new state names mean...

Reply:
I changed them in an attempt to be clearer and simpler. My thinking was that since this is the gatsby-source-filesystem plugin, that SRC_FS would indicate more clearly that this code was being executed inside this plugin vs a more core general purpose file watcher.

You are right though, I certainly don't know your reasons for choosing the names you have. There is a very likely chance that the names cannot/should not be changed. I was trying for the simplest way to achieve (what I believe) is the objective without being excessively verbose.

To decode:
- READY is triggered when chokidar fires it's ready event.
- SETUP is when BOOTSTRAP_FINISHED is fired.
- IDLE is when it is ready but is not processing
- BUSY is when it is processing a file

From my understanding (which granted I am 100% ok with being wrong) ;) there are just a couple events which this plugin needs to be aware of. I'm fairly certain that chokidar's ready event occurs before bootstrap is finished, but I tried to accommodate either order.

The other events that it needs to be concerned about are: is it processing a file currently, or not.